### PR TITLE
US118249: tree filter bug fix test (no selected ids case)

### PR DESCRIPTION
**Pull request by johngwilkinson**
Quality Notes
* functional
  * e2e w. LMS
    * first load (data contains final grades): histogram appears
    * select a course with no final grades: message appears
    * select a course with some final grades: histogram appears
* security: n/a
* perf: n/a
* devops: n/a
* ux/docs: approved via Slack

### CODE REVIEW

**Review by bemailloux — APPROVED** [review]

File: components/current-final-grade-card.js [hunk 1]
```diff
@@ -66,10 +75,22 @@ class CurrentFinalGradeCard extends Localizer(MobxLitElement) {
...
+					<span class="d2l-insights-empty-chart-message">
+						${this.localize('components.insights-current-final-grade-card.emptyMessage')}
+					</span>
+				</div>
+			</div>`;
+		} else {
+			return html`<div class="d2l-insights-final-grade-container">
+				<div class="d2l-insights-current-final-grade-title">${this._cardTitle}</div>
+				<d2l-labs-chart class="d2l-insights-summary-card-body" .options="${options}"></d2l-labs-chart>
+			</div>`;
```
[the remark this answers is not in the document]
Comment (johngwilkinson) [reply components/current-final-grade-card.js:92]: Host is a container. There's also a div around everything here.

File: components/current-final-grade-card.js [hunk 2]
```diff
@@ -66,10 +75,22 @@ class CurrentFinalGradeCard extends Localizer(MobxLitElement) {
...
 		// any change to a relevant observed property of the Data object
-		return html`<div class="d2l-insights-final-grade-container">
-		<div class="d2l-insights-current-final-grade-title">${this._cardTitle}</div>
-		<d2l-labs-chart class="d2l-insights-summary-card-body" .options="${this.chartOptions}"></d2l-labs-chart>
-		</div>`;
+		const options = this.chartOptions;
+		if (!options.series[1].data.length) {
+			return html`<div class="d2l-insights-final-grade-container">
+				<div class="d2l-insights-current-final-grade-title">${this._cardTitle}</div>
+				<div class="d2l-insights-summary-card-body">
```
[the remark this answers is not in the document]
Comment (johngwilkinson) [reply components/current-final-grade-card.js:82]: There's a story coming to handle this at the page level. This was just the quickest way for now.

This fix is specific to the histogram chart type.

File: components/current-final-grade-card.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (bemailloux) [reply components/current-final-grade-card.js:92]: But why? Seems like we could just do away with the extra container div

[the remark this answers is not in the document]
Comment (johngwilkinson) [reply components/current-final-grade-card.js:92]: Oh, I misunderstood your question. You're suggesting we get rid of the `d2l-insights-final-grade-container` div and style the host instead. We could. It was already done this way in this component, so I'm inclined to leave it.

**Review by anhill-D2L — APPROVED** [review]

**Pull request merged**